packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulators-info-display.jsx

```diff
@@ -15,31 +15,34 @@ const AccumulatorsInfoDisplay = observer(() => {
             label: localize('Max. payout'),
             value: <Money amount={maximum_payout} show_currency currency={currency} />,
             tooltip_text: localize('Your contract will be automatically closed when your payout reaches this amount.'),
+            margin: 143,
         },
         {
             label: localize('Max. ticks'),
             value: `${maximum_ticks || 0} ${maximum_ticks === 1 ? localize('tick') : localize('ticks')}`,
             tooltip_text: localize('Your contract will be automatically closed upon reaching this number of ticks.'),
+            margin: 175,
         },
     ];
 
     return (
         <Fieldset className={classNames('trade-container__fieldset', 'accu-info-display')}>
-            {content.map(({ label, value, tooltip_text }) => (
+            {content.map(({ label, value, tooltip_text, margin }) => (
                 <div key={label} className='accu-info-display__row'>
                     <Text size='xxs' weight='bold' line_height='xxs'>
                         {label}
                     </Text>
                     <Text size='xxs' align='right' as='div'>
-                        {value}
                         <Popover
-                            alignment='left'
-                            icon='info'
+                            alignment={isMobile() ? 'top' : 'left'}
+                            classNameBubble='accu-info-display__popover'
                             is_bubble_hover_enabled
                             message={tooltip_text}
-                            margin={isMobile() ? 0 : 216}
+                            margin={isMobile() ? -5 : margin}
```

Comment:
I know that negative margin - is a bag practice, but sometimes styling popover message for tooltip might be challenging. I tried to do with className, with classNameBubble but it didn't helped. If negative margin will be considered to be unacceptable, I'll remove even if it is a step of design.

Reply:
please check my 2 replies in this thread https://github.com/binary-com/deriv-app/pull/9676/files#r1298300155 
Might be related to this margin as well

Reply:
As for mobile this popover will go up ( `alignment={isMobile() ? 'top' : 'left'} `), this margin is regulating the space between bubble and tooltip itself (marked as red on the 1st screenshot). If we add margin-bottom to the classname, it wouldn't be applied to hole bubble, only part without small triangle (screenshot 2)
<img width="587" alt="Screenshot 2023-08-18 at 2 48 54 PM" src="https://github.com/binary-com/deriv-app/assets/121025168/e46b19ea-6d03-4df0-a8ea-8091227e4611">
<img width="1278" alt="Screenshot 2023-08-18 at 2 52 29 PM" src="https://github.com/binary-com/deriv-app/assets/121025168/1ed3ac9e-43c1-47f1-941a-25828a7fbc74">